pubspec.lock

```diff
@@ -1566,8 +1566,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: "twake-supported-0.22.4"
-      resolved-ref: d3fd4dbeb9ecfbee641753cd6267e27de9fe91f2
+      ref: fix_line_break_synax
```

Comment:
where is your PR
please attach in the issue description

Reply:
https://github.com/linagora/matrix-dart-sdk/pull/29/commits